pyproject.toml

```diff
@@ -68,11 +68,35 @@ archs = [
 ] # Forces arm64 build on x86_64 runner using cross-compilation.
 before-build = "export HOMEBREW_AUTO_UPDATING=0 && brew update && brew install ninja"
 
-[tool.pydocstyle]
-convention = "google"
-
 [tool.mypy]
 ignore_missing_imports = true
 
-[tool.isort]
-line_length = 120
+[tool.black]
+line-length = 120
+
+[tool.ruff]
+line-length = 120
+indent-width = 4
```

Comment:
Probably unnecessary, it is 4 by default, see https://docs.astral.sh/ruff/settings/#indent-width